source/StopWatch/Helpers/JiraTimeHelpers.cs

```diff
@@ -75,26 +76,32 @@ public static string TimeSpanToJiraTime(TimeSpan ts)
             if (time == "0")
                 return TimeSpan.Zero;
 
-            MatchCollection matches = new Regex(@"([0-9,\.]+[dhm] *?)+?", RegexOptions.IgnoreCase).Matches(time);
+            MatchCollection matches = new Regex(@"([-]?[0-9,\.]+[dhm] *?)+?", RegexOptions.IgnoreCase).Matches(time);
             if (matches.Count == 0)
                 return null;
 
             foreach (Match match in matches)
             {
                 s = match.Value.ToUpper();
                 s = s.Trim();
+                
+                if (s.StartsWith("-"))
+                    negative = true;
```

Comment:
Not sure about this.  You initialise negative = false _outside_ the loop, then update to true if that match is negative but don't reset for the enxt loop/match.  Suggest instead of this if statement just saying 
`negative = s.StartsWith("-")`
here?